.github/workflows/wcag_test.yml

```diff
@@ -6,8 +6,9 @@ on:
 
 jobs:
```

Comment:
like you mentioned in your comment, could we update this to run on feature branches too?

Reply:
yup! was gonna update it when I change back the workflow from dev branch to v1 in a bit